### ISSUE TITLE
How can I call a native from another plugin? I'm getting "Script not found" error.

### ISSUE THREAD

**Issue by KirillBorunov** [urShadow/RakNetManager#53]
I need to call natives of your plugin from another plugin.
But I'm getting a runtime error: "Script not found".

Calling natives to other plugins works good, but with this one not.

I'm using **sampgdk_FindNative(name)** to find your natives, this works OK.
But using **sampgdk_InvokeNativeArray(native, format, args)** not executes it and prints "[Pawn.RakNet] BS_ReadValue: Script not found" on the console. This happens with all your natives, not only the mentioned.

I'm doing something wrong?

**Comment by katursis** [comment]
No, sampgdk does something wrong. It invokes a native function using faked amx which Pawn.RakNet don't know about.

Calling once `AmxLoad` explicitly from Pawn.RakNet with `sampgdk_fakeamx_amx()` as argument may help you.

**Comment by dockfries** [comment]
Is there anyone who can help? it's been two years...

**Comment by dockfries** [comment]
I know almost nothing about sampgdk and the c language, and I was wondering if anyone could fix the situation, and not just raknet, but other plugin libraries like samp-cef can't be called through samp-node. The underlying samp-node should be the sampgdk_InvokeNativeArray that uses sampgdk.

I am willing to pay a certain amount of money to the person who solves the problem, in fact I am sponsoring open.mp, although only £18 GBP at the moment.
